rpm-ostree-container-bootc.ks.in

```diff
@@ -20,6 +20,15 @@ reboot
 
 %post
 
+curl fedoraproject.org
```

Comment:
That's not a URL so curl is probably going to always error. Obviously you could use `https://` in front to fix that, but since the goal is to test (just) DNS using e.g. `getent hosts fedoraproject.org` is more direct.

Reply:
On fedora there is systemd-resolved though.